tests/sklearn/test_sklearn_models.py

```diff
@@ -1923,16 +1929,21 @@ def test_rounding_consistency_for_regular_models(
 
 @pytest.mark.parametrize("model_class, parameters", get_sklearn_tree_models_and_datasets())
 @pytest.mark.parametrize("n_bits", [2, 5, 10])
-def test_fhe_sum_for_tree_based_models(
+@pytest.mark.parametrize("execution_mode", ["simulate", "fhe"])
```

Comment:
small detail but I find it more straigthforward to instead define a
```
@pytest.mark.parametrize("execute_in_fhe", [True, False])
```

Reply:
Yes will update.

Reply:
Done